README.md

```diff
@@ -131,7 +131,7 @@ Add the environment variable `JAVA_TOOL_OPTIONS` to your Lambda functions and se
 
 ### Supported Instrumentation Libraries
 
-- Aws SDK V1
+- Aws SDK V1 (Supported only from dependency and not from the auto trace with lambda layer)
```

Comment:
can we somehow know it and print it as warn?
I mean, if the sdk1 exist, and we were imported with a layer